scapy/contrib/lacp.py

```diff
@@ -0,0 +1,117 @@
+# This file is part of Scapy
+# Scapy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# any later version.
+#
+# Scapy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Scapy. If not, see <http://www.gnu.org/licenses/>.
+
+# scapy.contrib.description = LACP
+# scapy.contrib.status = loads
+
+from scapy.packet import Packet, bind_layers
+from scapy.fields import ByteField, MACField, ShortField, Field, ByteEnumField, IntField
+from scapy.layers.l2 import Ether
+from scapy.utils import lhex
+from scapy.data import ETHER_TYPES
+import struct
+
+
+ETHER_TYPES['SlowProtocol'] = 0x8809
+SLOW_SUB_TYPES = {
+    'Unused': 0,
+    'LACP': 1,
+    'Marker Protocol': 2,
+}
+
+
+class SlowProtocol(Packet):
+    name = "SlowProtocol"
+    fields_desc = [ByteEnumField("subtype", 0, SLOW_SUB_TYPES)]
```

Comment:
Could you please add spaces near the brackets:
`fields_desc = [ ByteEnumField("subtype", 0, SLOW_SUB_TYPES) ]`

Reply:
No please don't. That exists in Scapy but that's terrible. As mentioned in CONTRIBUTING.md, we're trying to conform to PEP-8 in new code (we don't want to change the existing code without another reason).

Reply:
my bad.
I've seen that so many times i thought it was the PEP8 format